### DISCUSSION TITLE
recipes/sides/cauliflower-pur%C3%A9e-with-thyme/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#248] [Comments] [answered]
# recipes/sides/cauliflower-pur%C3%A9e-with-thyme/

A collection of my recipes.

https://nicholaswilde.io/recipes/sides/cauliflower-pur%C3%A9e-with-thyme/

**Comment by nicholaswilde** [comment, marked as answer]
Used strainer in addition to slotted spoon to move the cooked cauliflower from the pot to the food processor. End result was quite runny just using the 3 Tbsp of liquid and cooking the 3 12-oz bags of TJ's cauliflower for the full 20 minutes. Perhaps next time can cook less time to get a thicker consistency.

Can also half butter amount because it already tastes "buttery".

**Reply by nicholaswilde** [reply]
Fixed by a13cfe9d65c7da6ed892a8de0a1a9cc566f0d9d1
